demo_gradle/plugins/plugin3/src/main/kotlin/ro/fortsoft/pf4j/demo/kotlin/KotlinPlugin.kt

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright 2017 Anindya Chatterjee
```

Comment:
`Copyright (C) 2017 the original author or authors.` ?

You are the author of this class so I don't see a problem with my propose. A copy/paste from an existed .java file is fine. Do you see a problem?